src/main/java/com/eprosima/fastcdr/idl/templates/FastCdrCommon.stg

```diff
@@ -66,6 +66,8 @@ $if(member.annotationDefault)$
 $elseif(!member.annotationOptional && !member.annotationExternal)$
 $if(member.typecode.initialValue)$
  {$member.typecode.initialValue$}
+$elseif(member.typecode.isBitsetType)$
+ {}
```

Comment:
Why is this left empty?

Reply:
The struct must be initialized. This fixes a windows warning.